torchtitan/experiments/vlm/model/model.py

```diff
@@ -76,7 +74,7 @@ def forward(
         tokens: torch.Tensor,
         pixel_values: torch.Tensor,
         grid_thw: torch.Tensor,
-        special_tokens: SpecialTokens,
+        img_id: int,
```

Comment:
nit: If we don't read the name `img_id` in the context of tokenizer, it might have other meanings. Previously we refer to it as `tokenizer.img_id` which should be fine, but here can we rename it to something like `img_token_id` in forward()